routers/web/repo/view.go

```diff
@@ -659,7 +650,7 @@ func checkHomeCodeViewable(ctx *context.Context) {
 
 			ctx.Data["Repo"] = ctx.Repo
 			ctx.Data["MigrateTask"] = task
-			ctx.Data["CloneAddr"] = safeURL(cfg.CloneAddr)
+			ctx.Data["CloneAddr"], _ = util.SanitizeURL(cfg.CloneAddr)
```

Comment:
```suggestion
			ctx.Data["CloneAddr"], _ = util.SanitizeURL(cfg.CloneAddr)
```
Any thoughts why we are suppressing the error aka `_`